docker/install_wordpress.sh

```diff
@@ -52,7 +52,7 @@ wp plugin install --activate --force \
     wp-rest-api-v2-menus \
     jwt-authentication-for-wp-rest-api \
     https://github.com/wp-graphql/wp-graphql/archive/master.zip \
-    https://github.com/wp-graphql/wp-graphql-jwt-authentication/archive/v0.3.1.zip \
+    https://github.com/wp-graphql/wp-graphql-jwt-authentication/archive/master.zip \
```

Comment:
I'm guessing this is where you ran into problems you had to debug? I'm thinking rather than pointing to `master`, which will remain a moving target, it might be good to just point at the current latest version? Otherwise if `master` introduces a breaking change in the future, the kit could break without any code changes on our end.

Reply:
the issue was that the wp-graphql is already on master, and the wp-graphql-jwt-authentication is on 0.3.1, so when the wp-graphql got updated, it break the authentication, so to keep both working correctly we need to use the same logic on both.
so it is either we keep both on master and they get updated together, or we need to change the wp-graphql to have a fixed version number.

Reply:
I vote to put wp-graphql on a fixed version, but @alexiakl probably knows best

Reply:
Magid, let's put both `wp-graphql-jwt-authentication` and `wp-graphql` on a fixed version, that should solve the breaking changes issues (also those two seem to be dependent on each other).
We are updating this repo relatively frequently, so we will bump up those libraries manually.

Reply:
👍